src/main/java/com/monumental/models/Monument.java

```diff
@@ -49,6 +51,7 @@
     @Column(name = "address")
     private String address;
 
+    @JsonIgnore
```

Comment:
Tags or images (not sure which) was causing an error when calling the API endpoint. This prevents them from being included in the JSON serialization, we will need to make another endpoint to get them separately (which I think is good practice because if we don't do this then the `getAll` endpoint will also return all these related objects which is probably not a good idea if there's a lot of them)